eel/__init__.py

```diff
@@ -145,8 +150,12 @@ def _eel():
 
 @btl.route('/<path:path>')
 def _static(path):
-    return btl.static_file(path, root=root_path)
-
+    if _env != None:
+        template = _env.get_template(path[10:])
```

Comment:
@sanders41 What's the `10:` for here?

Reply:
The setup assumes that templates will be placed inside a directory called "templates". The 10: removed templates/ from the path leaving only the name of the template.

Reply:
OK cheers. I will probably tweak it to make the folder name configurable